modules/ducktests/tests/ignitetest/services/utils/ignite_config.py

```diff
@@ -17,71 +17,49 @@
 This module renders Ignite config and all related artifacts
 """
 
+from jinja2 import FileSystemLoader, Environment
+
+import os
+
+DEFAULT_CONFIG_PATH = os.path.dirname(os.path.abspath(__file__)) + "/config"
+DEFAULT_IGNITE_CONF = DEFAULT_CONFIG_PATH + "/ignite.xml.j2"
+
+
+class Config(object):
```

Comment:
IgniteConfig?

Reply:
It has nothing Ignite specific. Just render any jinja2 templare.

Reply:
the filename is ignite_config, but the class name is just config.

Reply:
there are descenders: IgniteServerConfig and IgniteClientConfig that use ignite config files. Config class could render any config